lib/dpl/provider/packagecloud.rb

```diff
@@ -88,6 +88,7 @@ def get_source_files_for(orig_filename)
       end
 
       def push_app
+        forced = options.fetch(:force) { nil }
```

Comment:
Since we don't use the key value in the block, it is simpler to say:

```ruby
forced = options.fetch(:force, nil)
```

In fact, since the absence of the `:force` key in the `options` hash results in `nil`, you can also write:

```ruby
forced = options[:force]
```